src/app/icd/server/ICDManager.cpp

```diff
@@ -666,6 +668,14 @@ CHIP_ERROR ICDManager::HandleEventTrigger(uint64_t eventTrigger)
     case ICDTestEventTriggerEvent::kRemoveActiveModeReq:
         SetKeepActiveModeRequirements(KeepActiveFlag::kTestEventTriggerActiveMode, false);
         break;
+#if CHIP_CONFIG_ENABLE_ICD_CIP
```

Comment:
When is this not enabled?

Reply:
For SIT devices that still use the TestEventTrigger but might not have the Check-In Protocol support.